testnet/stacks-node/src/nakamoto_node/relayer.rs

```diff
@@ -1563,28 +1623,33 @@ impl RelayerThread {
     }
 
     /// Generate and submit the next block-commit, and record it locally
-    fn issue_block_commit(
-        &mut self,
-        tip_block_ch: ConsensusHash,
-        tip_block_bh: BlockHeaderHash,
-    ) -> Result<(), NakamotoNodeError> {
+    fn issue_block_commit(&mut self) -> Result<(), NakamotoNodeError> {
         if self.fault_injection_skip_block_commit() {
             warn!("Relayer: not submitting block-commit to bitcoin network due to test directive.");
             return Ok(());
         }
+        let (tip_block_ch, tip_block_bh) = SortitionDB::get_canonical_stacks_chain_tip_hash(
+            self.sortdb.conn(),
+        )
+        .unwrap_or_else(|e| {
+            panic!("Failed to load canonical stacks tip: {e:?}");
+        });
         let mut last_committed = self.make_block_commit(&tip_block_ch, &tip_block_bh)?;
 
-        // last chance -- is this still the stacks tip?
-        let (cur_stacks_tip_ch, cur_stacks_tip_bh) =
-            SortitionDB::get_canonical_stacks_chain_tip_hash(self.sortdb.conn()).unwrap_or_else(
-                |e| {
-                    panic!("Failed to load canonical stacks tip: {e:?}");
-                },
-            );
-
-        if cur_stacks_tip_ch != tip_block_ch || cur_stacks_tip_bh != tip_block_bh {
+        // last chance -- is this tenure still the canonical tip?
```

Comment:
Does this check still make sense? We just grabbed the tip.

Reply:
Yeah, I think this can be dropped.

Reply:
Removed in a6af0c1818cd0e5f4a56d1df20e1156d2909b27f